sdk/workloads/Azure.ResourceManager.Workloads/api/Azure.ResourceManager.Workloads.netstandard2.0.cs

```diff
@@ -0,0 +1,1742 @@
+        public override string ToString() { throw null; }
+    }
+    public partial class EnqueueReplicationServerProperties
+    {
+        public EnqueueReplicationServerProperties() { }
+        public Azure.ResourceManager.Workloads.Models.EnqueueReplicationServerType? ErsVersion { get { throw null; } }
+        public Azure.ResourceManager.Workloads.Models.SapHealthState? Health { get { throw null; } }
+        public string Hostname { get { throw null; } }
+        public string InstanceNo { get { throw null; } }
+        public string IPAddress { get { throw null; } }
+        public string KernelPatch { get { throw null; } }
+        public string KernelVersion { get { throw null; } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct EnqueueReplicationServerType : System.IEquatable<Azure.ResourceManager.Workloads.Models.EnqueueReplicationServerType>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public EnqueueReplicationServerType(string value) { throw null; }
+        public static Azure.ResourceManager.Workloads.Models.EnqueueReplicationServerType EnqueueReplicator1 { get { throw null; } }
+        public static Azure.ResourceManager.Workloads.Models.EnqueueReplicationServerType EnqueueReplicator2 { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.Workloads.Models.EnqueueReplicationServerType other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.Workloads.Models.EnqueueReplicationServerType left, Azure.ResourceManager.Workloads.Models.EnqueueReplicationServerType right) { throw null; }
+        public static implicit operator Azure.ResourceManager.Workloads.Models.EnqueueReplicationServerType (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.Workloads.Models.EnqueueReplicationServerType left, Azure.ResourceManager.Workloads.Models.EnqueueReplicationServerType right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class EnqueueServerProperties
+    {
+        public EnqueueServerProperties() { }
+        public Azure.ResourceManager.Workloads.Models.SapHealthState? Health { get { throw null; } }
+        public string Hostname { get { throw null; } }
+        public string IPAddress { get { throw null; } }
+        public long? Port { get { throw null; } }
+    }
+    public partial class ErrorDefinition
```

Comment:
Remove `Definition` suffix and probably add a prefix like `Workloads`?

Reply:
Rename to `SapVirtualInstanceErrorDetail`